android/java/org/chromium/chrome/browser/crypto_wallet/activities/AssetDetailActivity.java

```diff
@@ -44,7 +47,10 @@
 import org.chromium.chrome.browser.crypto_wallet.adapters.WalletCoinAdapter;
 import org.chromium.chrome.browser.crypto_wallet.listeners.OnWalletListItemClick;
 import org.chromium.chrome.browser.crypto_wallet.model.WalletListItemModel;
+import org.chromium.chrome.browser.crypto_wallet.observers.ApprovedTxObserver;
+import org.chromium.chrome.browser.crypto_wallet.observers.EthTxServiceObserver;
 import org.chromium.chrome.browser.crypto_wallet.observers.KeyringServiceObserver;
+import org.chromium.chrome.browser.crypto_wallet.util.PendingTxHelper;
```

Comment:
`EthTxServiceObserver` and `PendingTxHelper` imports are not needed

Reply:
yes, I am just wondering if it is possible to a lint check to remove the unused imports during build or `npm run lint` etc.

Reply:
it's a good question, @samartnik do you know?

Reply:
I'm not aware of such tools

Reply:
let's discuss to see the possibilities to add a java lint checker